caiman/source_extraction/cnmf/params.py

```diff
@@ -950,6 +950,11 @@ def check_consistency(self):
                 logging.warning(key + "=0, hence setting key update_num_comps " +
                                 "in group online automatically to False.")
 
+        # if using greedy init, ensure rf > gSiz
+        if (self.patch["rf"] is not None) and (self.init["gSiz"] is not None):
+            if not self.patch["rf"] > max(self.init["gSiz"]):
```

Comment:
Using >= rather than not > may make this more legible?

Reply:
oops this should not have gotten into this PR